### PR TITLE
71537 output bower versions

### PR DESCRIPTION
**Pull request by delta62**
@park9140 , there is no pullquestor data for this repo, but you're the only person I was going to send this to for code review.

### CODE REVIEW

File: tasks/lib/bowerTask.js [hunk 1]
```diff
@@ -81,6 +88,22 @@ BowerTask.prototype = {
...
+
+    printVersionTree: function (pkg, depth) {
+        this.grunt.log.writeln('-> ' + pkg.pkgMeta.name + '@' + pkg.pkgMeta.version);
+        for (var dep in pkg.dependencies) {
+            this.grunt.log.write(new Array(depth + 2).join('  '));
+            this.printVersionTree(pkg.dependencies[dep], depth + 1);
+        }
+    },
+
     install: function (options, retryLimit) {
```
Comment (park9140) [inline tasks/lib/bowerTask.js:107]: We need to make the version output run on install either here or in the build process.  Then we also need to create a quick test branch of ppmspa using this version of grunt-bower-tasks, to verify that it is being executed correctly in our teamcity build process.

**Comment by park9140** [comment]
After we run the tests in the build process we can pull this.

File: tasks/lib/bowerTask.js (hunk 1 above)
Comment (delta62) [inline tasks/lib/bowerTask.js:107]: It's already in the build process, PR #412.
I'll go fork the other repo for the TC build.

**Comment by delta62** [comment]
Awwwww yeahhhhh

http://teamcity/viewLog.html?buildId=483646&buildTypeId=Spa_CheckinGate&tab=buildLog#_focus=2863

**Pull request merged**